portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin:0;
+}
+
+#fullpage {
+  min-width: 100vw;
+  min-height: 100vh;
+  min-width: 100vw;
+  min-height: 100vh;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.section {
+  min-height: 100vh;
```

Comment:
Please list CSS properties in alphabetic order:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order
(here and in other groupsets)

Reply:
Corrected in commit 8eecc65

Reply:
I believe this CSS property section is still not sorted alphabetically, e.g., `max-height` should precede`min-height`; `overflow` should be last, etc.

Reply:
I've gone ahead and re-attempted the alphabetization with a [tool](https://www.freeformatter.com/css-beautifier.html) and have pushed this with commit e0eff1e